index.html

```diff
@@ -2,18 +2,86 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+    <meta charset="utf-8">
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <title>Great Idea!</title>
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--[if lt IE 9]>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
 </head>
-
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+    <header>
+        <nav>
+            <a href="#">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </nav>
+        <a class="logo-anchor" href="#">
+            <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
+    </header>
+
+    <section class="splash">
+        <div class="splash-text">
+            <h1>Innovation<br>On<br>Demand</h1>
+            <div class="start-button">
+                <a href="#">Get Started</a>
+            </div>
+        </div>
+        <img class="splash-img" src="img/header-img.png" alt="Image of a code snippet.">
+    </section>
+
+    <section class="top-section">
+        <div class="two-column">
+            <h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+
+        <div class="two-column">
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+    </section>
+
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  
+    <section class="bottom-section">
+        <div class="three-column">
+            <h2>Services</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+  
+        <div class="three-column">
```

Comment:
nice job reusing classes here for elements that you want to have the same styles!